src/utils/mod.rs

```diff
@@ -36,3 +38,89 @@ pub(crate) fn report_error(err: &anyhow::Error) {
         log::error!("{:?}", err);
     }
 }
+
+#[derive(strum::IntoStaticStr)]
+#[strum(serialize_all = "snake_case")]
+pub enum ConfigName {
+    RustcVersion,
+    LastSeenIndexReference,
+    QueueLocked,
+}
+
+pub fn set_config(
+    conn: &mut Client,
+    name: ConfigName,
+    value: impl Into<serde_json::Value>,
```

Comment:
Rather than `impl Into<Value>` this could be `impl Serialize` and use [`serde_json::to_value`](https://docs.rs/serde_json/latest/serde_json/fn.to_value.html), that would support nested types if we ever want to store something more than a simple string/bool in the future (like the previous `release_activity` data).

Reply:
I wasn't sure about the best API for these two methods, but you're right, `impl Serialize` could be the better one, both ways. 

I'll update it later today

Reply:
I updated the methods with an idea, I'm happy to take any feedback!